sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -264,6 +265,21 @@ export class Application extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   currentAssessment?: StudentAssessment;
+  /**
+   * All supporting users related to the application.
+   * These users (parents/partner) will be created as needed during
+   * the execution of the original assessment workflow processing.
+   */
+  @OneToMany(
```

Comment:
Is this a soft relationship, we have just for the execution of the query?

Reply:
As we talked, yes this one was added to allow the query. If needed an update/insert needs to be performed using this property the _cascade_ can be adapted.